frontend/components/organisms/Paginate/index.tsx

```diff
@@ -139,14 +139,14 @@ const Paginate = ({
 
     const renderIconItem = (item: PageItemType): JSX.Element => {
         return (
-            <li>
+            <li className="flex items-center">
                 <div
                     onClick={item.eventCallback}
-                    className={`ml-[4px] flex h-[24px] w-[24px] ${
+                    className={` ml-[4px] flex h-[16px] w-[16px] ${
```

Comment:
```suggestion
                    className={`ml-1 flex h-4 w-4 ${
```

You may refer to the tailwind docs for their equivalent classes. Please apply to all related.